libs/ui-lib/lib/common/components/clusterWizard/types.ts

```diff
@@ -34,6 +34,7 @@ export type ClusterDetailsValues = {
   platform: PlatformType;
   customOpenshiftSelect: OpenshiftVersionOptionType | null;
   userManagedNetworking: boolean;
+  controlPlaneCount: string;
```

Comment:
Out of curiosity, why not making it a integer ?

Reply:
by default the values of the form are strings, is better to transform when we send to the service